automated install/basic-install.sh

```diff
@@ -669,7 +669,7 @@ installScripts() {
 }
 
 installConfigs() {
-  # Install the configs from /etc/.pihole to their various locations
+  # Install the configs from ${PI_HOLE_LOCAL_REPO} to their various locations
```

Comment:
why even comment, just put it in the echo a couple lines lower and be more descriptive:

```
    echo "::: Installing configs from ${PI_HOLE_LOCAL_REPO}..." 
```

the rest of the echos describe 'various locations'